### PR TITLE
Disabled Creation of Multiple Loadouts for α Release

### LINKED ISSUE #1389: New Loadout + button hide
Please can you hide this button from the my games page as loadouts are not ready. 

Image

### PR DESCRIPTION
**Pull request by Sewer56**
fixes #1389

This disables the creation of more than 1 loadout for the α Release.

image

The pill has been modified so the `View` button stretches to the size of the remaining space.

Post first α release, we can make the setting visible in Release builds.
(Currently this setting is Debug only and does not ship to users)

image

image

The pill with `Add` button restored.

-------------

Outside of that, unsupported games have been hidden from Release builds by default and the features have very clearly been marked as `Experimental` since we don't yet have visible Settings categories.

I also renamed the `manage-game` CLI command, since it was inaccurate with what it did given the recent changes. Somehow that slipped past review.

### CODE REVIEW

**Comment by halgari** [comment]
Oh gosh, I'm going to be pedantic on this again. Didn't @Pickysaurus mention we wanted "Unsupported" not "Experimental"? Or was I not listening?

**Comment by Sewer56** [comment]
> Oh gosh, I'm going to be pedantic on this again. Didn't @Pickysaurus mention we wanted "Unsupported" not "Experimental"? Or was I not listening?

The response was in reference as to what to name the setting to enable all games. Which is called `Enable Unsupported Games`.

Because we don't have setting categories, I pretended text to the dangerous ones.

I could make.it so it says `(Unsupported) Enable Unsupported Games` though, even if it looks a bit weird.

File: src/NexusMods.App.UI/Controls/GameWidget/IGameWidgetViewModel.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public interface IGameWidgetViewModel : IViewModelInterface
     public ReactiveCommand<Unit,Unit> RemoveAllLoadoutsCommand { get; set; }
 
     public GameWidgetState State { get; set; }
+    public bool CanAddMoreThanOneLoadout => false;
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/GameWidget/IGameWidgetViewModel.cs:20]:
```suggestion
    public bool CanAddMoreThanOneLoadout { get; }
```

File: src/NexusMods.App.UI/Controls/GameWidget/GameWidgetViewModel.cs [hunk 2]
```diff
@@ -77,4 +80,5 @@ public GameWidgetViewModel(ILogger<GameWidgetViewModel> logger)
 
     [Reactive]
     public GameWidgetState State { get; set; }
+    public bool CanAddMoreThanOneLoadout => _experimentalSettings.EnableMultipleLoadouts;
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/GameWidget/GameWidgetViewModel.cs:83]:
```suggestion
    public bool CanAddMoreThanOneLoadout { get; }
```

File: src/NexusMods.App.UI/Controls/GameWidget/GameWidgetViewModel.cs [hunk 3]
```diff
@@ -13,10 +14,12 @@ namespace NexusMods.App.UI.Controls.GameWidget;
 public class GameWidgetViewModel : AViewModel<IGameWidgetViewModel>, IGameWidgetViewModel
 {
     private readonly ILogger<GameWidgetViewModel> _logger;
+    private readonly ExperimentalSettings _experimentalSettings;
 
-    public GameWidgetViewModel(ILogger<GameWidgetViewModel> logger)
+    public GameWidgetViewModel(ILogger<GameWidgetViewModel> logger, ISettingsManager settingsManager)
     {
         _logger = logger;
+        _experimentalSettings = settingsManager.Get<ExperimentalSettings>();
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/GameWidget/GameWidgetViewModel.cs:22]:
```suggestion
        CanAddMoreThanOneLoadout = settingsManager.Get<ExperimentalSettings>().EnableMultipleLoadouts;
```

File: src/NexusMods.App.UI/Controls/GameWidget/GameWidgetViewModel.cs [hunk 4]
```diff
@@ -13,10 +14,12 @@ namespace NexusMods.App.UI.Controls.GameWidget;
 public class GameWidgetViewModel : AViewModel<IGameWidgetViewModel>, IGameWidgetViewModel
 {
     private readonly ILogger<GameWidgetViewModel> _logger;
+    private readonly ExperimentalSettings _experimentalSettings;
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/GameWidget/GameWidgetViewModel.cs:17]:
```suggestion
```

**Review by erri120 — CHANGES_REQUESTED** [review]

**Review by erri120 — APPROVED** [review]

**Pull request merged**